storybook/src/utils/VisuallyHidden/VisuallyHidden.stories.tsx

```diff
@@ -3,25 +3,16 @@
  * Copyright Gemeente Amsterdam
  */
 
-import { Paragraph, VisuallyHidden } from '@amsterdam/design-system-react/src'
 import { Meta, StoryObj } from '@storybook/react'
+import type { HTMLAttributes } from 'react'
+
+type VisuallyHiddenProps = HTMLAttributes<HTMLSpanElement>
+
+const VisuallyHidden = ({ children }: VisuallyHiddenProps) => <span className="ams-visually-hidden">{children}</span>
```

Comment:
The code view doesn't render well this way.

Reply:
It does now, through the `render` prop of a Story.

Reply:
If you are going to create an example why not use the react components and add `className='ams-visually-hidden'` to the Paragraph component. 

And render is not needed to achieve this, you could use `parameters.docs.source.code` to add a static code example.

Reply:
The public React component gets deleted in this PR. The stories file has a local component instead because a Story requires it.

I updated the example to use React `Paragraph` components, even though these CSS components are usable outside of React as well, e.g. in our future libraries. But for now, mixin React here seems fine.

The render function seems a more realisatic proof of how this class name works than a fragment of static code, or am I missing something? Can you propose an example?